backend/api/migrations/0056_alter_computetask_status.py

```diff
@@ -0,0 +1,51 @@
+# Generated by Django 4.2.9 on 2024-02-13 14:38
+
+from django.db import migrations
+from django.db import models
+
+
+def migrate_data(apps, schema_editor):
+    status_mapping = {
+        "STATUS_WAITING": "STATUS_WAITING_FOR_PARENT_TASKS",
+        "STATUS_TODO": "STATUS_WAITING_FOR_EXECUTOR_SLOT",
+    }
+    model_model = apps.get_model("api", "computetask")
+    for task_instance in model_model.objects.all():
+        new_status = status_mapping.get(task_instance.status_old, task_instance.status_old)
+        task_instance.category = new_status
+        task_instance.save()
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("api", "0055_function_image_address_function_image_checksum"),
+    ]
+
+    operations = [
+        migrations.RenameField(
+            model_name="computetask",
+            old_name="status",
+            new_name="status_old",
+        ),
+        migrations.AddField(
+            model_name="computetask",
+            name="status",
+            field=models.CharField(
+                choices=[
+                    ("STATUS_WAITING_FOR_BUILDER_SLOT", "Status Waiting For Builder Slot"),
+                    ("STATUS_BUILDING", "Status Building"),
+                    ("STATUS_WAITING_FOR_PARENT_TASKS", "Status Waiting For Parent Tasks"),
+                    ("STATUS_WAITING_FOR_EXECUTOR_SLOT", "Status Waiting For Executor Slot"),
+                    ("STATUS_DOING", "Status Doing"),
+                    ("STATUS_DONE", "Status Done"),
+                    ("STATUS_CANCELED", "Status Canceled"),
+                    ("STATUS_FAILED", "Status Failed"),
```

Comment:
Did you tried to deploy a frontend to see how it reacts ? I'm afraid that the new name are too long and might lead to a weird behaviour :/

Reply:
We are going to address that with product and design tomorrow or beginning next week and do a follow-up PR